### PR TITLE
Implement cookie for theme choice

### PR DESCRIPTION
**Pull request by DavidMatalik**
Peek 2021-06-25 14-59

### CODE REVIEW

**Review by RichStone — CHANGES_REQUESTED** [review]
where did you copy-paste get- and setCookie from? :D (I don't like how they named things, it makes the boiler code way less readable. It's now part of your codebase, so I'd change the namings ;))

getCookie and setCookie functions are great candidates for `lib`-like functionality. (functionality that doesn't belong to any component really and that's potentially reusable across the applicaiton)
wanna try to add a `src/lib` folder to your application and extract these functions there?

File: src/App.js [hunk 1]
```diff
@@ -17,7 +17,32 @@ class App extends Component {
       foo: "bar",
       resumeData: {},
       sharedData: {},
+      mode: (this.getCookie('mode') === 'true') ? true : false
```
Comment (RichStone) [inline src/App.js]: would this be enough? 
```js
mode: this.getCookie('mode') === 'true'
```

**Comment by DavidMatalik** [comment]
> getCookie and setCookie functions are great candidates for `lib`-like functionality. (functionality that doesn't belong to any component really and that's potentially reusable across the applicaiton)
> wanna try to add a `src/lib` folder to your application and extract these functions there?

Yeaaaahh I forgot about it... You've mentioned it already. ;) And yes it felt a bit wrong to put it there. I will change it as requested.

**Comment by DavidMatalik** [comment]
> where did you copy-paste get- and setCookie from? :D (I don't like how they named things, it makes the boiler code way less readable. It's now part of your codebase, so I'd change the namings ;))

From w3schools. ;) I'll change the naming.

Edit: You are right, I really had to figure out what is going on in the cookie functions. I hope it is a bit easier to understand now.

File: src/App.js (hunk 1 above)
Comment (DavidMatalik) [reply src/App.js]: Ah yes, you are right =)

**Review by RichStone — APPROVED** [review]

**Comment by RichStone** [comment]
coool!

**Pull request merged**